File: sdk/src/main/java/com/cxense/cxensesdk/model/UserExternalDataResponse.java
package com.cxense.cxensesdk.model;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.List;

/**
 * Response for data associated with the user(s) from server.
 *
 * @author Dmitriy Konopelkin (dmitry.konopelkin@cxense.com) on (2017-06-05).
 */
public final class UserExternalDataResponse {
    @JsonProperty("data")
    public List<UserExternalData> items;
}

File: sdk/src/main/java/io/piano/android/cxense/model/UserExternalDataResponse.kt
package io.piano.android.cxense.model

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

/**
 * Response for data associated with the user(s) from server.
 *
 */
@Deprecated("See UserExternalTypedDataResponse")
@JsonClass(generateAdapter = true)
public class UserExternalDataResponse(
    @Json(name = "data") public val items: List<UserExternalData> = emptyList(),
)
